content/docs/utilities/z-index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a specified `z-{value}` class to your element to change the order of the stack. Remember to specify at least `position: relative` on the element for `z-index` to work.
```

Comment:
```suggestion
You can use these utility classes to control the stack. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```

Is there another term other than "the stack"? It's weird to me to have that repeated

Reply:
Not sure...

Reply:
maybe something like:

> You can use these utility classes to control **elements layered in a stack**. Add a `z-{value}` class to your element to change the order of the stack.`

Explains what "the stack" is referencing.